LICENSE.md

```diff
@@ -1,6 +1,6 @@
 The UnitfulIntegration.jl package was authored by Andrew Keller and is licensed under the MIT "Expat" License:
 
-> Copyright (c) 2017, California Institute of Technology. All rights reserved.
+> Copyright (c) 2022, California Institute of Technology. All rights reserved.
```

Comment:
```suggestion
> Copyright (c) 2017-2022, California Institute of Technology. All rights reserved.
```
I would either keep `2017` or use the range `2017-2022`. But it doesn’t really matter.